packages/server/test/node.spec.ts

```diff
@@ -386,6 +375,129 @@ describe('Node Specific Cases', () => {
           expect(response.status).toBe(200);
           await expect(response.text()).resolves.toBe('Hello world!');
         });
+
+        it('responds with the correct status code', async () => {
```

Comment:
Should each STATUS_CODE assertion be a single test case instead?

```
Object.entries(STATUS_CODES).forEach(statusCode => {
  it(`... ${statusCode}`, async () => {})
})
```

Reply:
It was like that before but then the log becomes hard to read. Imagine all status codes for every single server etc.